tox.ini

```diff
@@ -5,15 +5,6 @@ envlist =
 [testenv]
 deps =
     zope.testing
+    subunit: python-subunit
```

Comment:
[Factors](https://testrun.org/tox/latest/config.html#factors-and-factor-conditional-settings) are parts of environment names.  This line makes `py27-subunit`, `py34-subunit`, and `pypy-subunit` valid environments, with that dependency installed.